docs/settings.rst

```diff
@@ -103,6 +103,18 @@ of ``mozilla-django-oidc``.
 
    The OpenID Connect scopes to request during login.
 
+.. py:attribute:: OIDC_STORE_ACCESS_TOKEN
+
+   :default: ``False``
+
+   If true, store the access_token in ``session['oidc_access_token']``.
```

Comment:
I think this needs work. This sentence only says what it does which might be a good start but it really ought to flesh out a bit more about *why* and *why not* this is something you'd want to set.